test/ToneAnalyzer.v3.UnitTests/ToneAnalyzerServiceGeneratedUnitTests.cs

```diff
@@ -118,7 +107,6 @@ public void Tone_Success()
             JObject bodyObject = new JObject();
             var json = JsonConvert.SerializeObject(bodyObject);
             request.Received().WithArgument("version", versionDate);
-            request.Received().WithBodyContent(Arg.Is<StringContent>(x => x.ReadAsStringAsync().Result.Equals(json)));
```

Comment:
Need to check why these are not working. I suspect because of the change in signature.

Reply:
it works with 
`request.Received().WithBodyContent(Arg.Is<StringContent>(x => x.ReadAsStringAsync().Result.Equals("")));`